serverless-aws-typescript/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+import * as apigateway from "@pulumi/aws-apigateway";
+
+// A Lambda function to invoke
+const fn = new aws.lambda.CallbackFunction("fn", {
+    callback: async (ev, ctx) => {
```

Comment:
Given how often folks hork themselves using callback functions -- and this is painful to suggest because I love them -- should we have code that uses archives, like with the other languages? In addition to less horkability, it's also symmetric across examples. Perhaps you could have one of each so they can pick the style that works best?

Reply:
Maybe - though I believe the "magic" of this is one of the most exciting things we can highlight for the serverless use case in TypeScript.  (Lots of feedback supporting this was the reason for getting interested in Pulumi).  

Perhaps the README for this could be the place to note the alternative approach?

Reply:
I'm going to keep this as is for now, but it's something we can revisit in the future and/or add to Next Steps in the marketing pages.